packages/x-components/src/components/decorators/injection.consts.ts

```diff
@@ -52,3 +52,10 @@ export const SELECTED_VARIANTS_KEY: XInjectKey<ResultVariant[]> = 'selectedVaria
 export const SELECT_RESULT_VARIANT_KEY: XInjectKey<
   (variant: ResultVariant, level?: number) => void
 > = 'selectResultVariant';
+
+/**
+ * It's used to identify filters without facetId.
+ *
+ * @internal
+ */
+export const FACET_KEY: string | number = '__unknown__';
```

Comment:
If I used here `XInjectKey<string | number>`it forced typecasting `FACET_KEY` in several places, and could not manage to make it work properly inside [filter-entity.factory.ts](https://github.com/empathyco/x/pull/1351/files#diff-05dba8b269de2cb9ed3550cc44c32f9b6e50bf8318d97e56d31d13a9f4f3d9c4), so I left it like this.